README.md

```diff
@@ -59,25 +59,30 @@ You will need the following groups of dependencies to run the project successful
    - [.NET Core v3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1)
    - [Azure Functions Core Tools v3+](https://github.com/Azure/azure-functions-core-tools)
    
-2. To emulate azure storage you will require either:
+2. To emulate azure storage you will require:
    - [Azure Storage Emulator](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator)
 
-  To run Azure Storage Emulator, you'll need to install LocalDB via SQL Express. You can find the installer for Windows x64 [here](https://download.microsoft.com/download/3/9/F/39F968FA-DEBB-4960-8F9E-0E7BB3035959/SQLEXPR_x64_ENU.exe).
+  To run Azure Storage Emulator, you'll need to install LocalDB via SQL Express. You can find the installer for Windows 
+  x64 [here](https://download.microsoft.com/download/3/9/F/39F968FA-DEBB-4960-8F9E-0E7BB3035959/SQLEXPR_x64_ENU.exe).
 
-   - [Azurite](https://github.com/Azure/Azurite)
    - [Azure Storage Account](https://azure.microsoft.com/en-gb/services/storage/) 
    - [LocalDB](https://download.microsoft.com/download/2/A/5/2A5260C3-4143-47D8-9823-E91BB0121F94/ENU/x64/SqlLocalDB.msi)
    
-   You will most likely need to use Windows to run the service due to a dependency on Azure Storage Emulator
-for development, specifically the table storage component which is not supported by [Azurite](https://github.com/Azure/Azurite). However this does not apply to every component of the service.
+   You will currently need to use Windows or a Windows VM with ports 10001, 10002 and 10003 exposed to run the service 
+   due to a dependency on Azure Storage Emulator for development, specifically the table storage component which is not 
+   supported by [Azurite](https://github.com/Azure/Azurite). However this does not apply to every component of the 
+   service.
 
-   Alternatively you could create your own Storage Account on Azure and ammend your storage connection string to point to this.
+   Alternatively you could create your own Storage Account on Azure and amend your storage connection strings to point 
+   to this.
 
 3. To run the databases, you can use either:
 
    - [SQL Server 2017+](https://www.microsoft.com/en-gb/sql-server/sql-server-downloads)
    - [Docker and Docker Compose](https://docs.docker.com/)
 
+The SQL Server Developer Edition is free and more fully-featured than SQL Express. Note that SQL Express is still 
+required for its LocalDB support for the Azure Storage Emulator.
```

Comment:
We could recommend initialising storage explorer with SQL server rather than SQL Express as mentioned above, to avoid people needing to install that too.